corehq/form_processor/abstract_models.py

```diff
@@ -37,6 +37,10 @@ def set_submission_properties(self, submission_post):
     def to_json(self):
         raise NotImplementedError()
 
+    @property
+    def is_normal(self):
```

Comment:
it's simple in the basic case:
```
0: archived
1: error
2: duplicate
3: deprecated
4: submission_error_log
```
but now if it has multiple states, the solution at the previous place i worked was using bit masking. what if the form is a submission_error_log, a duplicate, and an error? you're right though in saying it's like the `doc_type` property and i don't think we actually worry about that scenario currently